tests/unit/test-util/test.json.js

```diff
@@ -0,0 +1,27 @@
+import {it, describe} from 'mocha';
+import {assert} from 'chai';
+import {fs} from 'mz';
+
+import {parseJSON} from '../../../src/util/json';
+import {basicManifest} from '../helpers';
+
+
+describe('util.parseJSON', () => {
+
+  it('returns a parsed JSON', () => {
+    const parsedJSON = parseJSON(JSON.stringify(basicManifest));
```

Comment:
You will get a Flow error here because you aren't passing enough arguments to `parseJSON`. I think it would be better to test this with a temp file. You can see how it's done here: https://github.com/mozilla/web-ext/blob/master/tests/unit/test-util/test.manifest.js#L53-L69